src/main/java/sirius/biz/charts/SpiderChart.java

```diff
@@ -0,0 +1,352 @@
+        });
+
+        return addDataset(dataset);
+    }
+
+    /**
+     * Sets the marks for the chart that are rendered as small ticks on the axes.
+     *
+     * @param marks the marks to set
+     * @return the chart itself for fluent method calls
+     * @see #withRings()
+     */
+    public SpiderChart<N> withMarks(List<N> marks) {
+        this.marks.clear();
+        this.marks.addAll(marks);
+        this.marks.sort(Comparator.comparingDouble(Number::doubleValue));
+        return this;
+    }
+
+    /**
+     * Enables drawing of rings around the chart, connecting the equivalent marks of all the axes.
+     *
+     * @return the chart itself for fluent method calls
+     * @see #withMarks(List)
+     */
+    public SpiderChart<N> withRings() {
+        this.rings = true;
+        return this;
+    }
+
+    /**
+     * Sets a formatter for the numeric values of the marks.
+     *
+     * @param formatter the formatter to use
+     * @return the chart itself for fluent method calls
+     */
+    public SpiderChart<N> withFormatter(Function<N, String> formatter) {
+        this.formatter = formatter;
+        return this;
+    }
+
+    public SequencedSet<String> getLabels() {
+        return Collections.unmodifiableSequencedSet(labels);
+    }
+
+    public List<Dataset<N>> getDatasets() {
+        return Collections.unmodifiableList(datasets);
+    }
+
+    @Override
+    public Element toSvg(Dimension bounds) {
+        Element svgElement = createSvgElementWithCenteredViewbox(bounds);
+
+        Element backgroundGroupElement = createGroupElement(svgElement.getOwnerDocument());
+        svgElement.appendChild(backgroundGroupElement);
+
+        Element axesGroupElement = createGroupElement(svgElement.getOwnerDocument());
+        backgroundGroupElement.appendChild(axesGroupElement);
+
+        Element labelsGroupElement = createGroupElement(svgElement.getOwnerDocument());
+        backgroundGroupElement.appendChild(labelsGroupElement);
+
+        Element graphsGroupElement = createGroupElement(svgElement.getOwnerDocument());
+        svgElement.appendChild(graphsGroupElement);
+
+        double radius = 0.5 * Math.min(bounds.width, bounds.height) - 7.5;
+        double normaliser = computeNormaliser();
+
+        // draw scale with circles and labels
+        drawAxes(svgElement.getOwnerDocument(), radius, normaliser, axesGroupElement, labelsGroupElement);
+
+        AtomicInteger datasetCounter = new AtomicInteger();
+        datasets.forEach(dataset -> {
+            int datasetIndex = datasetCounter.getAndIncrement();
+            StringBuilder pathDefinition = new StringBuilder();
+
+            dataset.withColor(Dataset.COLORS.get(datasetIndex % Dataset.COLORS.size()));
+
+            AtomicInteger labelCounter = new AtomicInteger();
+            labels.forEach(label -> {
+                int labelIndex = labelCounter.getAndIncrement();
+
+                double radians = 2.0 * Math.PI * labelIndex / labels.size();
```

Comment:
**Optional:** Here no constant for tau is used :) Which is also present in the Java standard library as `Math.TAU`